diff --git a/vendor/rust-errno/Cargo.toml b/vendor/rust-errno/Cargo.toml
index 6a5b3f8..88e100b 100644
--- a/vendor/rust-errno/Cargo.toml
+++ b/vendor/rust-errno/Cargo.toml
@@ -15,13 +15,6 @@ rust-version = "1.56"
 [target.'cfg(unix)'.dependencies]
 libc = { version = "0.2", default-features = false }
 
-[target.'cfg(windows)'.dependencies.windows-sys]
-version = "0.52"
-features = [
-  "Win32_Foundation",
-  "Win32_System_Diagnostics_Debug",
-]
-
 [target.'cfg(target_os="wasi")'.dependencies]
 libc = { version = "0.2", default-features = false }
 
